### PR TITLE
Updated judge-worker submodule to master version

### PR DESCRIPTION
**Pull request by gogo4ds**
Update judge-worker submodule to latest stable version
closes https://github.com/SoftUni-Internal/suls-issues/issues/5201

### CODE REVIEW

File: Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        }
+
+        public static void DecryptAppConfigSections(params string[] sectionNames)
+        {
+            var config = GetExeConfiguration();
+
+            UnprotectConfigurationSections(config, GetSections(config, sectionNames));
+        }
+
+        private static Configuration GetExeConfiguration() =>
```
Comment (achobanov) [inline Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs]: Shouldn't arrow be on next line?

File: Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+
+        private static IEnumerable<ConfigurationSection> GetSections(
+            Configuration config,
+            params string[] sectionNames) =>
```
Comment (achobanov) [inline Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs]: Shouldn't arrow be on next line?

File: Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs (hunk 1 above)
Comment (gogo4ds) [reply Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs]: in OJS this is the convention :/

File: Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs (hunk 2 above)
Comment (gogo4ds) [reply Open Judge System/OJS.Common/Helpers/EncryptionHelper.cs]: in OJS this is the convention :/

**Pull request merged**